main.c

```diff
@@ -423,23 +420,23 @@ int main(int argc, char *argv[]) {
   state.sem = dispatch_semaphore_create(1);
   iface = start(&state, cliopt);
   if (iface == NULL) {
-    perror("start");
+    ERRORN("start");
     goto done;
   }
 
   if (pid_fd != -1) {
     char pid[20];
     snprintf(pid, sizeof(pid), "%u", getpid());
     if (write(pid_fd, pid, strlen(pid)) != (ssize_t)strlen(pid)) {
-      perror("pidfile_write");
+      ERRORN("pidfile_write");
```

Comment:
Should be ERRNO("write"), but using ERRORF with more context will be more helpful.

Reply:
fixed to ERRORN("write") - more context for ERRORN can be added later.